### PR TITLE
Updated version number to 4.3.13 for sauce-labs

### PR DESCRIPTION
**Pull request by iugo-robert**

### CODE REVIEW

File: Casks/sauce-connect.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask :v1 => 'sauce-connect' do
-  version '4.3.6'
+  version '4.3.13'
   sha256 '9a678e751658cc207237a7d5aec87aabe4684143c04bf23983f627f0605c9d3b'
```
Comment (vitorgalvao) [inline Casks/sauce-connect.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by iugo-robert** [comment]
I've also updated the SHA256 line as per your instructions.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by iugo-robert** [comment]
Closed this one, see other one I created.

**Pull request closed without merging**